src/view/use-lift-instruction/use-lift-instruction.js

```diff
@@ -24,8 +23,8 @@ export default function useLiftInstruction(
       // add the description text
       el.textContent = liftInstruction;
 
-      // hide the element visually
-      Object.assign(el.style, visuallyHidden);
```

Comment:
Ah okay. So there are some things we want visually hidden, but available to the screen readers. And there are other things we want visually hidden + not available to screen readers

Reply:
Yeah if we reference it only via something like `aria-describedby` then it doesn't need to be visible. The `visuallyHidden` is mainly for things like icon buttons (using `aria-label` is better), and I believe it's needed for live regions too.